packages/react-router-permissions/README.md

```diff
@@ -0,0 +1,202 @@
+# Authorization
+
+## Installation
+
+Using [yarn](https://yarnpkg.com/lang/en/):
+
+    $ yarn add @tshio/react-router-permissions
+
+Using [npm](https://www.npmjs.com/):
+
+    $ npm install --save @tshio/react-router-permissions
+
+## Usage
+
+Goal of this package is to provide abstraction layer for handling authorization with react-router.
+The only requirement for `AuthorizedRoute` to work is to have any kind `Router` and at least one `PermissionsProvider`
+higher in your component tree.
+
+```js
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
+
+const store = {
+  authorization: {
+    roles: ['USER', 'ADMIN'],
+  },
+};
+
+const authorizationStrategy = (roles, requirement) => {
+  return roles.find(role => role === requirement);
+};
+
+// it's possible to override strategy for single route
+const loginAuthorizationStrategy = (roles, requirement) => {
+  return roles && roles.length;
+};
+
+class App extends React.Component {
+  render() {
+    return (
+      <Provider store={store}>
+        <PermissionsProvider
+          permissions={store.getState().authorization.rolses}
```

Comment:
store is an object in this context, i get this is an example, but it might be confusing to someone